.vscode/settings.json

```diff
@@ -51,22 +51,22 @@
     "dist/": true,
     "doc/": true,
     "node_modules/": true,
-    "package/": true,
+    "package/": true
   },
   "protoc": {
     "options": [
-      "--proto_path=source_relative"  // https://github.com/zxh0/vscode-proto3/issues/31#issuecomment-628162495
-    ],
+      "--proto_path=source_relative" // https://github.com/zxh0/vscode-proto3/issues/31#issuecomment-628162495
+    ]
   },
   "alignment": {
     "operatorPadding": "right",
-    "indentBase":      "firstline",
-    "surroundSpace":   {
-      "colon":      [1, 1],   // The first number specify how much space to add to the left, can be negative. The second number is how much space to the right, can be negative.
-      "assignment": [1, 1],    // The same as above.
-      "arrow":      [1, 1],    // The same as above.
-      "comment":    2,         // Special how much space to add between the trailing comment and the code.
-                               // If this value is negative, it means don't align the trailing comment.
```

Comment:
We are trying to add spaces to align between lines for values and comments.

Does prettier support the above rule?